dpub-aam/role/roles.html

```diff
@@ -0,0 +1,76 @@
+<!doctype html>
+<html>
+<head>
+  <title>Simple DPUB-AAM Role Verification Tests</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<p>DPUB-AAM changed expectations for role description and other details since the initial v1.0 implementations, many of which are not yet testable via WPT, but this tests the new <a href="https://w3c.github.io/dpub-aam/#mapping_role_table">DPUB `computedrole` mapping</a> expectations.</p>
```

Comment:
```suggestion
<p>DPUB-AAM changed expectations for role description and other details since the initial v1.0 implementations, many of which are not yet testable via WPT, but this tests the new <a href="https://w3c.github.io/dpub-aam/#mapping_role_table">DPUB `computedrole` mapping</a> expectations. Additional rationale in <a href="https://github.com/w3c/aria/issues/1887">ARIA #1887</a>.</p>
```